ui-test/.env

```diff
@@ -0,0 +1,40 @@
+#
```

Comment:
For consistency with backend e2e tests please rename this folder to `test/smoke`.

Reply:
The ui-test folder is at the 'top level', as a sibling to the ui folder because both are Node projects.  Moving this to test/smoke will mean these tests will be `mixed in` with the go test cases at a lower level.  Should we want to keep those tests homogeneous?